src/main/java/uk/gov/companieshouse/charges/data/model/ChargesDocument.java

```diff
@@ -1,20 +1,30 @@
 package uk.gov.companieshouse.charges.data.model;
 
 import java.util.Objects;
-import org.bson.codecs.pojo.annotations.BsonProperty;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Version;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
 import uk.gov.companieshouse.api.charges.ChargeApi;
 
+@Document(collection = "company-mortgages")
```

Comment:
Please externalise the collection name in applicatio yml so that we can override name using an env variable